conbench/job.py

```diff
@@ -80,6 +86,8 @@ def _fetch_and_cache_most_recent_results(n=0.07 * 10**6) -> None:
     by_case_dict: Dict[str, List[BenchmarkResult]] = defaultdict(list)
     for result in results:
         by_id_dict[result.id] = result
+        # Add property _hardware on result obj, from Run.
+        result._hardware = result.run.hardware
```

Comment:
And this is just lazy JP who wants to be _sure_ that the next access to `result.run` does not issue a db query. This needs to be cleaned up.